frontend/logfile.lua

```diff
@@ -0,0 +1,71 @@
+--[[--
+This module helps with writing information to log files
+]]
+
+local LogFile = {
+    log_files = {},
```

Comment:
As mentionned, not certain we need to keep this module.
It does in some more expensive way (but ok, in a Lua portable way), what we do in the startup shell script on some platforms:
https://github.com/koreader/koreader/blob/9f37863f00f518239ac5420afdeb118ab525ff68/platform/kobo/koreader.sh#L376-L380

It is named "logfile", but we don't (and won't) use it for our main "log file", our crash.log, so it would deserve a more modest name.
If we were to use it for more "active" log files, we would probably need to optimise it (ie. don't rewrite on each new line added, but rather remove 10% of ilnes so we won't have to do that for the next additions. And for less "active" log files like the version log, we don't really need shrinkage.